src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,383 @@ tree_item_index: 1100
+When using the manual transition, the user uploads content to the transfer bucket. While in this step, a user can add folders and files to the bucket and add properties that apply to the files. Properties can be added that apply to all files using the “common metadata” panel; more file-specific properties can be added either to multiple files at once (bulk edit of metadata) and/or individually,  using a single-file action.
+Once the metadata has been edited to fit the user's needs, the transfer can be **completed** and the content is then created in the system.
+Transfers have 3 states: Draft (no content added yet), In Progress (content added) and Completed (the transfer has been completed and the content has been created in the system).
+
+
+#### Using Aspera with Nuxeo
+
+Once in your instance, Nuxeo Aspera Upload can be accessed from two different ways:
+
+- From the **User Settings** menu located on the left side:
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image3
+    name: aspera-image3.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image3](nx_asset://2a6ee847-e886-4047-8e82-f4cafda627eb ?border=true)
+
+- By clicking on the Aspera upload button displayed on every folderish document (workspace, folder, etc.)
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image19
+    name: aspera-image19.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image19](nx_asset://da4443e9-0ddb-4908-9ce8-5fcce2e108ce ?border=true)
+
+> The files uploaded by Aspera will be accessible in this folderish document (by default the target location is the user personal workspace)
+
+Once on the Aspera Upload screen (as you begin to add files), you will be able to download the Aspera Desktop client via the following banner at the top (if not already installed or if a new version is needed):
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image14
+    name: aspera-image14.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image14](nx_asset://304c7c51-ec27-4a37-9b02-439879412d6a ?border=true)
+
+Once on the Aspera menu, there are 2 main views available:
+
+- [Transfers Dashboard](#transfers-dashboard)
+
+- [Transfers](#transfers)
```

Comment:
To avoid confusion, should we rename to something like Aspera Dashboard and Transfer View / Details ?

Reply:
@andreacornaglia good idea.  we'll need to do something since, as we saw in the nike meeting, we'll want to really document what actions do and where they take you.